### ISSUE TITLE
Plugin works but no highlighting

### ISSUE THREAD

**Issue by kecerud** [folke/todo-comments.nvim#148]
Hey there! I have installed this plugin via `Plug` and didn't set anything besides the basic config like so:

In `init.vim`

```
Plug 'nvim-lua/plenary.nvim'
Plug 'folke/todo-comments.nvim'
```

In the lua setup file:

```
require("todo-comments").setup {}
```

And while the plugin seems to work since it shows the icons, it doesn't highlight correctly whatsoever:

<img width="198" alt="Screenshot 2022-10-28 at 13 42 16" src="https://user-images.githubusercontent.com/75565018/198579231-f0ea0f41-8a3b-42e7-893e-58f7419ffee2.png">

I'm using [gruvbox](https://github.com/morhetz/gruvbox) as my colortheme and I tried disabling it but without any luck..

Note: the example code on the screenshot is written in a `.js` file but the highlighting doesn't work in any other file type for example the `init.vim` itself.

**Comment by folke** [comment]
Do you have the JavaScript treesitter parser installed?

**Comment by folke** [comment]
comment detection is done through treesitter, so you need treesitter, or disable in comments only

**Comment by kecerud** [comment]
@folke 

1) :My treesitter config: 

<img width="1058" alt="Screenshot 2022-10-28 at 14 42 33" src="https://user-images.githubusercontent.com/75565018/198589376-e80cf507-6dec-4485-a7ae-f9b24914a8e3.png">

Is there anything wrong? 🤔

**Comment by kecerud** [comment]
@folke I see, but that's weird, after `:TSUpdate` it says that:

<img width="353" alt="Screenshot 2022-10-28 at 15 04 35" src="https://user-images.githubusercontent.com/75565018/198597451-781e9e6d-3b87-4b81-a338-c4b2cdbc30c7.png">

Interesting :D

**Comment by folke** [comment]
What does the following say in that buffer?

```vim
:lua print(vim.treesitter.highlighter.active[vim.api.nvim_get_current_buf()])
```

**Comment by kecerud** [comment]
It printed this: 

<img width="207" alt="Screenshot 2022-10-28 at 15 09 41" src="https://user-images.githubusercontent.com/75565018/198600118-670926c6-ee10-4ab0-a871-2681141ef187.png">

**Comment by folke** [comment]
ok, that is odd.

Can you install https://github.com/nvim-treesitter/playground

Then go over a comment and execute `:TSHighlightCapturesUnderCursor`

**Comment by kecerud** [comment]
<img width="512" alt="Screenshot 2022-10-28 at 15 15 22" src="https://user-images.githubusercontent.com/75565018/198602821-6726322f-5174-455a-9cf7-6a5cfbcd8a84.png">

**Comment by folke** [comment]
It looks like you have syntax enabled in that buffer as well, which is not ideal.

Probably won't fix it, but try setting `additional_vim_regex_highlighting=false` in your treesitter config.

It's never a good idea to have that enabled anyway, since it leads to alls sorts of issues.

**Comment by kecerud** [comment]
Thanks for the tip, after disabling there's only the `Treesitter` highlighting left 🤔 

<img width="305" alt="Screenshot 2022-10-28 at 15 30 23" src="https://user-images.githubusercontent.com/75565018/198611681-9ecccf08-4941-457e-bdc1-2b43ae0337c3.png">

**Comment by folke** [comment]
```vim
: lua   vim.pretty_print(vim.treesitter.get_captures_at_pos(vim.api.nvim_get_current_buf(), vim.fn.row(".") - 1, vim.fn.col(".") - 1))
```

**Comment by kecerud** [comment]
Not sure if it's the right command or somethings broken on my side

<img width="587" alt="Screenshot 2022-10-28 at 15 46 46" src="https://user-images.githubusercontent.com/75565018/198621576-845310b8-86b8-424b-9996-85053288a386.png">

**Comment by folke** [comment]
Should be `line`, my bad

**Comment by kecerud** [comment]
<img width="566" alt="Screenshot 2022-10-28 at 16 23 50" src="https://user-images.githubusercontent.com/75565018/198642860-7eb5f691-a4ac-478d-8450-a9f90a4e0056.png">

**Comment by folke** [comment]
That;s all really odd. `comment` is there, so it shold be working

**Comment by folke** [comment]
Pretty sure it's because of the recent treesitter changes that basically made all treesitter themes not working anymore.

Every theme has to be fixed to be able to use the newest nvim-treesitter

**Comment by kecerud** [comment]
You're right!!! Wow I actually didn't thought of trying this just with your `tokyonight` theme :D That's hilarious.

Thanks for the help! It works completely fine with your theme , in fact i'll try to get used to it and in the meantime I'll try to hit up the creator of gruvbox if he can update the theme for this. Is there any specific changes that need to be added to the theme so I can point him to it right away or?

**Comment by folke** [comment]
I think it' because of Neovim 0.8.0. The Diagnostics groups are now baked in, so always exist, but probably dont have default styling, so my fallback groups wont trigger

**Comment by folke** [comment]
Good idea

**Comment by nighi** [comment]
I have exactly this issue with gruvbox and neovim v0.9.5. It works with the tokyonight theme. How could I configure these hl-group colors for gruvbox? 
Already tried with
```lua
opts={
    colors = {
        error = { '#DC2627' },
        warning = { '#FBBF26' },
        info = { '#2563EB' },
        hint = { '#10B981' },
        default = { '#7C3AED' },
        test = { '#FF00FF' },
    },
}
```
but that changed nothing.
